cvat-ui/src/components/email-confirmation-page/email-confirmed.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React, { useRef } from 'react';
+import { Link } from 'react-router-dom';
+import { Col, Row } from 'antd/lib/grid';
+import Layout from 'antd/lib/layout';
+import Statistic from 'antd/lib/statistic';
+import './styles.scss';
+
+const { Content } = Layout;
+const { Countdown } = Statistic;
+const deadline = Date.now() + 1000 * 6;
+
+/**
+ * Component for displaying email confirmation message and then redirecting to the login page
+ */
+
+function EmailConfirmationPage() {
+    const linkRef = useRef();
+    const onFinish = () => {
+        linkRef.current.click();
+    };
+    return (
```

Comment:
Just my thoughts how to do the component more reliable. Right now deadline is computed only once when the file is imported. In theory in case this page opened several times, it will not work as expected. In spite of the fact that this scenario is not expected to appear in real, I suggest followings:

```ts
    const [deadline, setDeadline] = useState<null | number>(null);
    useEffect(() => {
        setDeadline(6);
    }, [])
    return (
        deadline === null ? null : (
            ... current return code ...
        )
    )
```

What do you think?

Reply:
Or even simpler option probably will work:

```
<Countdown format='ss' title='Redirecting to login page after...' value={Date.now() + 1000 * 6} onFinish={onFinish} />
```

Reply:
> Or even simpler option probably will work:
> 
> ```tsx
> <Countdown format='ss' title='Redirecting to login page after...' value={Date.now() + 1000 * 6} onFinish={onFinish} />
> ```

yes, better and simpler